cdap-ui/app/hydrator/controllers/create/toppanel-ctrl.js

```diff
@@ -343,19 +343,26 @@ class HydratorPlusPlusTopPanelCtrl {
       _cdapNsPath: '/previews/' + previewId + '/status',
       interval: 5000
     }, (res) => {
-      if (res.status !== 'RUNNING') {
+      if (res.status !== 'RUNNING' && res.status !== 'STARTED') {
         this.stopTimer();
         this.previewRunning = false;
         this.dataSrc.stopPoll(res.__pollId__);
+        let pipelineName = this.HydratorPlusPlusConfigStore.getName();
+        let pipelinePreviewPlaceholder = pipelineName.length > 0 ? `Preview of pipeline "${pipelineName}"` : 'Pipeline preview';
         if (res.status === 'COMPLETED') {
           this.myAlertOnValium.show({
             type: 'success',
-            content: 'Pipeline preview is finished.'
+            content: `${pipelinePreviewPlaceholder} is finished.`
           });
-        } else {
+        } else if (res.status === 'STOPPED' || res.status === 'KILLED') {
           this.myAlertOnValium.show({
             type: 'success',
-            content: 'Pipeline preview was stopped successfully.'
+            content: `${pipelinePreviewPlaceholder} was stopped successfully.`
```

Comment:
Hm I think the passive tense makes sense here, since the preview was stopped because of the action of an user. 

Actually it might make more sense to the change `${pipelinePreviewPlaceholder} is finished.` to just `${pipelinePreviewPlaceholder} finished.`? Since it finishes by itself.